extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -86,9 +101,9 @@ Implicit tiling also allows for better interoperability with existing GIS data f
 
 Implicit tiling enables procedurally-generated tilesets. Instead of serving static files, a server could extract the tile coordinates from [Template URIs](#template-uris) and generate tiles at runtime while using little disk space.
```

Comment:
Technically procedural generation could be done before on a server by parsing content uris, it was just more ad-hoc. It's not a super strong use case so maybe we should just remove this paragraph.